infra/modules/database/authentication.tf

```diff
@@ -38,6 +41,8 @@ data "aws_iam_policy_document" "app_db_access" {
 
     resources = [
       "${local.db_user_arn_prefix}/${var.app_username}",
+      # TODO: Remove migrator username. Part 3 of multipart update https://github.com/navapbc/template-infra/issues/354#issuecomment-1693973424
+      "${local.db_user_arn_prefix}/${var.migrator_username}",
```

Comment:
doing it this way will break things and defeats the purpose of us doing the db layer changes first, since this requires you to apply in db layer first before applying in service layer, but we have CD that autodeploys the service.

i think you need to do [the idea i suggested in this PR comment](https://github.com/navapbc/platform-test/pull/38#discussion_r1308076465):

> what if we attached the migrator access policy to the app_service role rather than to the migrator role in the migrator_db_access resource below? then we can remove this policy attachment.

in other words, don't change the policies that were created in the db layer. just make sure you attach both the app access policy and the migrator access policy to the service.